pkg/conform/export.go

```diff
@@ -199,35 +199,27 @@ func ExportChartDirectory(chart *chart.Chart, targetPath string) error {
 	if err != nil {
 		return err
 	}
+	defer os.RemoveAll(tempDir)
```

Comment:
As it is, yes. This was a conscious decision I made, but I'm not sure it's the right decision. We could handle the error, but because it is a deferred function, we would have to give `ExportChartDirectory` a named return value like this:
```
func ExportChartDirectory(chart *chart.Chart, targetPath string) (err error) {
```
We could then set `err` in an anonymous deferred wrapper function, which itself would call `os.RemoveAll(tempDir)`.

But I figured that the chances of an error in the `os.RemoveAll` call are very low, because we are removing a directory that we created in the first place - I can't think of a reason why this would fail. And if it does fail, we are left with a directory that `git status` will tell us about, and which is easily removed.

I'm happy to change it if you want me to, but it will make this function somewhat harder to understand.